### PR TITLE
Extend td04test ref6

### PR DESCRIPTION
**Pull request by repagh**
Extended the test cases with the case in issue #6

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -21,6 +21,13 @@ requirements:
    # this link needed quite some searching, please do not delete!
    # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
 
+  build:
```
Comment (murrayrm) [inline conda-recipe/meta.yaml]: This build section is repeated from above.  Should be deleted, right?

Comment (repagh) [reply conda-recipe/meta.yaml]: Right. Don't know how that got cloned. Checking locally and then committing.

**Comment by murrayrm** [comment]
Can you pull out the `meta.yml` comments since those overlap with PR #31?  (Also, not related to main change here.)

While you are at it, perhaps squash the commits into a single commit, to keep it cleaner?  (We can also do that on merge, if it is a hassle.)

**Comment by repagh** [comment]
abandoning this mess of commits!

**Pull request closed without merging**